Description: Add the missing junit dependency (inherited from the ignored parent pom)
Author: Emmanuel Bourg <ebourg@pache.org>
Forwarded: not-needed
--- a/openssl-dynamic/pom.xml
+++ b/openssl-dynamic/pom.xml
@@ -219,4 +219,13 @@
       </build>
     </profile>
   </profiles>
+
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+    </dependency>
+  </dependencies>
+
 </project>
